logexport/config.py

```diff
@@ -0,0 +1,19 @@
+import os
+from typing import Final, Mapping
+
+ADDITIONAL_LABEL_PREFIX: Final[str] = "ADDITIONAL_LABEL_"
+
+
+def get_additional_labels() -> dict[str, str]:
+    """Returns a dictionary of additional labels to add to the exported streams.
+    The labels are configured through environment variables."""
+    return get_additional_labels_from_env(os.environ)
+
+
+def get_additional_labels_from_env(env: Mapping[str, str]) -> dict[str, str]:
```

Comment:
nit: actually this could be called `get_additional_labels_from_mapping`, since it consumes a map, and it's not coupled with the env itself.